test/ValidityTest.hs

```diff
@@ -111,31 +110,31 @@ operationStripDir = do
                 stripDir parent (parent </> child) == Just child
 
      it "produces a valid path on when passed a valid absolute file paths" $ do
-        producesValidsOnValids2 (stripDir @Maybe @Abs @File)
+             producesValidsOnValids2 (stripDir :: Path Abs Dir -> Path Abs File -> Maybe (Path Rel File))
```

Comment:
I don't mind the extra indentation but I'm wondering whether there's a system behind it?

Most projects I've seen indent with either 2 or 4 spaces in a more or less consistent way.